staging/src/k8s.io/apiserver/pkg/storage/interfaces.go

```diff
@@ -73,16 +73,13 @@ type ResponseMeta struct {
 	ResourceVersion uint64
 }
 
-// MatchValue defines a pair (<index name>, <value for that index>).
-type MatchValue struct {
-	IndexName string
-	Value     string
-}
+// TriggerPublisherFunc is a function that for a given object computes
+// <value of an index> for a particular <index>.
+type TriggerPublisherFunc func(obj runtime.Object) string
```

Comment:
Can we rename this to something like `IndexerFunc`? I didn't find the current name helpful in understanding what the function does (same for `TriggerPublisherFuncs` below).

Reply:
Added a TODO - will fix in a subsequent PR.